content/en/real_user_monitoring/installation/advanced_configuration.md

```diff
@@ -18,6 +21,24 @@ Find below the different initialization options available with the [Datadog Brow
 
 By default, no sampling is applied on the number of collected sessions. To apply a relative sampling (in percent) to the number of sessions collected, use the `sampleRate` parameter when initializing RUM. The following example collects only 90% of all sessions on a given RUM application:
 
+{{< tabs >}}
+{{% tab "NPM" %}}
+
+```javascript
+import { datadogRum } from '@datadog/browser-rum'
+
+datadogRum.init({
+  applicationId: '<DATADOG_APPLICATION_ID>',
+  clientToken: '<DATADOG_CLIENT_TOKEN>',
+  datacenter: 'us',
+  resourceSampleRate: 100,
```

Comment:
`resourceSampleRate` is not documented for the bundle version, IMO we should be consistent between the two setup options. We could either remove it from the npm setup or add it to the bundle setup and add explanations for it.
@hdelaby any preferences?

Reply:
waiting on @hdelaby feedback on this to take action

Reply:
Let's delete it.